src/system/tests/TestSystemPacketBuffer.cpp

```diff
@@ -309,7 +309,11 @@ TEST_F_FROM_FIXTURE(TestSystemPacketBuffer, CheckNew)
     {
         const PacketBufferHandle buffer = PacketBufferHandle::New(0, config.reserved_size);
 
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
+        if (config.reserved_size > CHIP_CONFIG_MAX_LARGE_PAYLOAD_SIZE_BYTES)
```

Comment:
I see this pattern of ifdef repeated several times where we switch back and forth between either PacketBuffer::kMaxSizeWithoutReserve or CHIP_CONFIG_MAX_LARGE_PAYLOAD_SIZE_BYTES

How about having a single place where we define a constant of like `kMaxPacketBufferPayloadSize` or similar name and have this ifdef only once.